features/admin.remote-userstores.v1/components/edit/userstore-general-settings.tsx

```diff
@@ -517,354 +516,356 @@ export const UserStoreGeneralSettings: FunctionComponent<UserStoreGeneralSetting
-                                            required={ false }
-                                            placeholder="Enter the description of the user store"
-                                            maxLength={ 300 }
-                                            minLength={ 3 }
-                                            width={ 14 }
-                                            data-testid={ `${ testId }-user-store-description-textarea` }
-                                            value={
-                                                userStore?.description
-                                                    ? userStore?.description
-                                                    : ""
-                                            }
-                                            validation={ (value: string, validation: Validation) => {
-
-                                                let isMatch: boolean = true;
-                                                let validationErrorMessage: string;
-
-                                                const validityResult: Map<string, string | boolean> =
-                                                    validateInputWithRegex(value,
-                                                        USERSTORE_VALIDATION_REGEX_PATTERNS.EscapeRegEx);
-                                                const validationMatch: boolean = (
-                                                    validityResult.get("isMatch").toString() === "true");
-
-                                                if (validationMatch) {
-                                                    isMatch = false;
-                                                    const invalidString: string = validityResult.get(
-                                                        "invalidStringValue").toString();
-
-                                                    validationErrorMessage =
-                                                        t("console:manage.features.userstores.forms.general.description"
-                                                            + ".validationErrorMessages.invalidInputErrorMessage", {
-                                                            invalidString: invalidString
-                                                        });
-                                                } else {
-                                                    isMatch = true;
+            <EmphasizedSegment padded="very">
+                <Grid>
+                    <Grid.Row>
+                        <Grid.Column width={ 6 }>
+                            <Forms
+                                onSubmit={ (values: Map<string, FormValue>) => {
+                                    handleUpdateDescription(values);
+                                } }
+                            >
+                                <Field
+                                    requiredErrorMessage={ null }
```

Comment:
Can we avoid this prop or pass a meaningful value?

Reply:
Fixed 70f200f14255f037a46096205acd8f666919bb76